styles/index.css

```diff
@@ -0,0 +1,63 @@
+.main {
+  text-align: center;
+  color: #14213D;
```

Comment:
might be nice for us to have colours as variables in the future - but i am happy to do this as a cleanup task 🤓

Reply:
Agreed!